doc/getting-started.md

```diff
@@ -21,59 +19,72 @@ deployment "istio-mixer" created
 ## Connecting microservices with Istio
 
 This guide shows how to set up Istio and manipulate the proxy mesh to achieve useful behavior.
-In this example we use two microservices:
+In this example we use *CouchDB* and *NGINX* as and demonstrate how to manipulate them:
+
+* CouchDB is the official CouchDB Docker image
+* NGINX is the offical NGINX Docker image.  We will configure it to redirect all calls to CouchDB
 
-* microservice "hello" returns a constant bit of JSON.
-* microservice "frontend" calls microservice "hello"
+First, let's start CouchDB and have it join the Istio service mesh
 
-The communicating microservices are examples from Kubernetes'
-[Connecting a Front End to a Back End Using a Service](https://kubernetes.io/docs/tutorials/connecting-apps/connecting-frontend-backend/) tutorial.
+```
+# Start the official Docker CouchDB image the container in a Kubernetes Pod.  5984 is the CouchDB port.
+kubectl run couchdb --image=docker.io/library/couchdb --port 5984
+```
 
-First we will start the microservices using the Istio pattern of a
-proxy within frontend's pod. The `istioctl kube-inject` command
-injects the istio runtime proxy into kubernetes resource files. It is
-documented [here](istioctl.md#kube-inject).
+The _kubectl get deployment_ subcommand simply creates a machine readable description of a service.  The _istioctl kube-inject_ subcommand creates a machine readable description of a service including Istio components.  By piping the commands together and sending the output to _kubectl apply_ we add Istio capabilities to our service.
 
 ```
+# The CouchDB deployment must join the Istio service mesh.  Note that the pod will be recreated.
+kubectl get deployment couchdb --output yaml | istioctl kube-inject --filename - --tag 2017-03-22-17.30.06 | kubectl apply -f -
+```
 
-# Backend "hello" service with an instance of container gcr.io/istio-testing/runtime:demo
-kubectl create -f <(istioctl kube-inject -f doc/hello-and-proxy.yaml)
+For this tutorial an NGINX front-end will talk to CouchDB.  To enable communication we must expose the deployment as a Kubernetes Service.
 
-# "Frontend" service with an instance of container gcr.io/istio-testing/runtime:demo
-kubectl create -f <(istioctl kube-inject -f doc/frontend-and-proxy.yaml)
-kubectl expose -f <(istioctl kube-inject -f doc/frontend-and-proxy.yaml) --type=NodePort --name=frontend
+```
+kubectl expose deployment couchdb --name=couchdb
+# Let Istio know that the port is HTTP, not generic TCP
+kubectl patch service couchdb --type='json' -p='[{"op": "add", "path": "/spec/ports/0/name", "value":"http"}]'
```

Comment:
This command is applicable only when dealing with a live system. It would be better if its preceded by an example of k8s YAML snippet that shows how to set the port name attribute before launching the service.